content/en/docs/02/04_update.md

```diff
@@ -4,3 +4,198 @@ weight: 24
 sectionnumber: 2.4
 ---
 
+In this lab we will update our OpenShift 4 cluster to a newer minor version.
+
+Updating an OpenShift 4 cluster is a fully automated process that is managed by several ClusterOperators. If the cluster has Internet access, updates are provided over-the-air, otherwise you need to synchronize the new images to an internal registry prior to starting the update or use a pull-through registry.
```

Comment:
```suggestion
Updating an OpenShift 4 cluster is a fully automated process that is managed by several cluster operators. If the cluster has Internet access, updates are provided over-the-air, otherwise you need to synchronize the new images to an internal registry prior to starting the update or use a pull-through registry.
```